src/lib/network_pool/network_pool_base.ml

```diff
@@ -43,10 +43,52 @@ end)
     |> ignore ;
     network_pool
 
+  (* Rebroadcast locally generated pool items every 5 minutes or every slot,
+     whichever is slower. Do so for half an hour before giving up. *)
```

Comment:
Can you include some reasoning as to why we want to give up after half-an-hour? Why not do something with exponential backoff that goes on for much longer (a few hours; a day?). It seems only better to retry for longer up until the point where one would want to have their generated item expire

Reply:
done